mne/forward/_lead_dots.py

```diff
@@ -185,8 +186,12 @@ def _comp_sums_meg(beta, ctheta, lut_fun, n_fact, volume_integral):
 ###############################################################################
 # SPHERE DOTS
 
-def _fast_sphere_dot_r0(r, rr1, rr2, lr1, lr2, cosmags1, cosmags2,
-                        w1, w2, volume_integral, lut, n_fact, ch_type):
+_meg_const = 4e-14 * np.pi  # This is \mu_0^2/4\pi
+_eeg_const = 1.0 / (4.0 * np.pi)
+
+
+def _fast_sphere_dot_r0(r, rr1_orig, rr2s, lr1, lr2s, cosmags1, cosmags2s,
+                        w1, w2s, volume_integral, lut, n_fact, ch_type):
```

Comment:
@jasmainak see changes to this function for improved vectorization of computations